Dockerfile

```diff
@@ -79,6 +79,16 @@ RUN /bin/bash -c 'if [ "${USE_DOT_NET}" = "yes" ] ; then \
   mv /root/.dotnet/* /usr/local/dotnet; \
 fi'
 
+# if kubectl is set then install kubectl
+ARG USE_KUBECTL
+RUN /bin/bash -c 'if [ "${USE_KUBECTL}" = "yes" ] ; then \
+  curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"; \
```

Comment:
Make this architecture aware. X86-64 and ARM64. https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/

Like the install of GOLANG does below.